framework/docker/ibc/relayer/hermes.go

```diff
@@ -0,0 +1,479 @@
+package relayer
```

Comment:
this is the main implementation and is mostly a port of the logic introduced in interchaintest [here](https://github.com/strangelove-ventures/interchaintest/pull/396)